### PR TITLE
Always send sample rate for logs

### PR DESCRIPTION
**Pull request by trask**
Missed this in #38737

### CODE REVIEW

File: sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java [hunk 1]
```diff
@@ -139,6 +139,7 @@ public void map(SpanData span, Consumer<TelemetryItem> consumer) {
             consumer);
     }
 
+    // TODO looks like this method can be private
     public TelemetryItem map(SpanData span, long itemCount) {
```
Comment (heyams) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java:143]: this method is used in agent.

**Review by heyams — COMMENTED** [review]
can we fix classicSDK too?

File: sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java (hunk 1 above)
Comment (trask) [reply sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java:143]: can you show me where?

**Comment by trask** [comment]
> can we fix classicSDK too?

this needs to be fixed on the agent side

File: sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java (hunk 1 above)
Comment (heyams) [reply sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java:143]: [AgentSpanExporter line 62](https://raw.githubusercontent.com/microsoft/ApplicationInsights-Java/main/agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/AgentSpanExporter.java#:~:text=mapper.map(span%2C%20telemetryItemConsumer)%3B)

Comment (trask) [reply sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java:143]: that's not the same method

**Review by heyams — APPROVED** [review]

**Pull request merged**